### ISSUE TITLE
Exception thrown when using Geocoding.Microsoft

### ISSUE THREAD

**Issue by jphellemons** [chadly/Geocoding.net#88]
````
IGeocoder geocoder = new Geocoding.Microsoft.BingMapsGeocoder("my api key here");
IEnumerable<Address> addresses = geocoder.GeocodeAsync("amsterdam").Result;
````
Cannot convert type **Geocoding.Microsoft.Json.Resource** to type **Geocoding.Microsoft.Json.Location**

**Comment by jphellemons** [comment]
Strange, just tried to reproduce it by forking the repo. Have made a new console app and added a reference to both core and microsoft project. had to add system.net.http to the console app (nuget) and it all just works... without the nuget for system.net.http it gives an error about system.net.http 4.1.1.2

**Comment by jphellemons** [comment]
Updated all my Nuget packages for my web project and still have this error on line
````
foreach (Json.Location location in response.ResourceSets[0].Resources)
````
in Geocoding.Microsoft.BingMapsGeocoder in method ParseResponse

**Comment by 0657code** [comment]
Resources are of the types Location or Route. Since this API only supports Locations, there should be only Locations returned.

I suppose the NuGet-Package you used was outdated and did not contain the fix for #76. The response was serialized incorrectly, returning a Resource instead of a Location, and thus the cast failed.

**Comment by chadly** [comment]
There was some unreleased code. The fix is now in [`v4.0.1`](https://github.com/chadly/Geocoding.net/releases/tag/v4.0.1).

**Comment by jphellemons** [comment]
@chadly Great! you just updated the nuget 👍 https://www.nuget.org/packages/Geocoding.Microsoft/ Thank you and also @0657code 👍
